schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/activity/DefaultActivityService.java

```diff
@@ -704,6 +703,13 @@ public Key addActivityPackage(ActivityPackage activityPackage, List<Activity> ac
         Key organizationId = Preconditions.checkNotNull(activityPackage.getOrganizationRef().getKey());
         activities = Preconditions.checkNotNull(activities);
 
+        if (activityPackage.getItemCount() <= 0) throw new FieldValueException("ActivityPackage.itemCount");
+        if (activities.isEmpty() || activities.size() == 1 || activities.size() < activityPackage.getItemCount()) {
+            throw new FieldValueException("ActivityPackage.activities");
+        }
+        if (new HashSet<>(activities).size() != activities.size()) throw new FieldValueException("ActivityPackage.activities");
```

Comment:
Could you enrich the messages in the exceptions.  ActivityPackage.activites will confuse the frontend developer that faces these ;-)